Add `remove_token_from_body_when_cookies_used` config option (#940)
lexik/LexikJWTAuthenticationBundle: Resources/doc/1-configuration-reference.md

            cookies:
                - jwt_hp
                - jwt_s
                

>     # do not remove the token from the body when using cookies

Reviewer: ```suggestion
    # remove the token from the response body when using cookies
```
